cli/main.rs

```diff
@@ -405,7 +405,10 @@ fn resolve_flags_and_init(
         // TODO(petamoriken): Need to check TypeScript `assert` keywords in deno_ast
         vec!["--no-harmony-import-assertions".to_string()]
```

Comment:
It seems that this flag is no longer needed.

Reply:
Let's leave it for now as the TODOs here seem important.